views/account.jade

```diff
@@ -2,66 +2,75 @@ extends layout
 
 block content
   .container.main
-    h1= title
-    h2 Welcome,
-      = " "
-      if user.name_given
-        span.name= user.name_given
-      else
-        span.name= user.email
+    .row
+       .col-md-4.col-md-offset-1.text-left
+        img(src='/images/HID-BIG.png', alt='Error').hid-logo-dashboard
```

Comment:
@uiftikhar This is wrong, the image does not show up. You need to point it to /images/HID-big.png or /images/HID_BIG.png. Also, there are 2 images: HID-big.png and HID_BIG.png. Please remove one of them.